### DISCUSSION TITLE
开始抢票后显示有错误

### DISCUSSION THREAD

**Discussion by yuxiuaaa1** [mikumifa/biliTickerBuy#139] [提问专区]
![QQ截图20240619213711](https://github.com/mikumifa/biliTickerBuy/assets/139149361/6f98c311-9cee-4c67-9fb2-917bc8e1ac80)

**Comment by mikumifa** [comment]
时间忘记该了，马上发新版本修了

**Comment by mikumifa** [comment]
等v2.8.4
